{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-20143",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/about.gitlab.com\/blog\/categories\/releases\/",
                    "name": "https:\/\/about.gitlab.com\/blog\/categories\/releases\/",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/about.gitlab.com\/releases\/2020\/01\/02\/security-release-gitlab-12-6-2-released\/",
                    "name": "https:\/\/about.gitlab.com\/releases\/2020\/01\/02\/security-release-gitlab-12-6-2-released\/",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in GitLab Community Edition (CE) and Enterprise Edition (EE) 12.6. It has Incorrect Access Control."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-01-13T21:15Z",
    "lastModifiedDate": "2020-01-13T21:21Z"
}